webui/Caddyfile

```diff
@@ -5,6 +5,19 @@ redir 301 {
   / https://{host}{uri}
 }
 
+proxy /auth keycloak:8080 {
+  transparent
+}
+
+proxy /api api:80 {
+  without /api
+  transparent
+}
+
+proxy /api/files havenapi:3000 {
+  transparent
+}
+
```

Comment:
These are updates for the production caddyfile to try and catch up with the dev configuration